### DISCUSSION TITLE
Is qT-XT compatible with new WP 6.5 performant translations?

### DISCUSSION THREAD

**Discussion by Peter-HVD** [qtranslate/qtranslate-xt#1405] [General]
So today's release of WordPress 6.5 has brought the new Performant Translations system in to core.

https://make.wordpress.org/core/2024/02/27/i18n-improvements-6-5-performant-translations/

Do we know yet whether qT-XT has any issues with this?

Cheers

Peter

**Comment by Komarovski** [comment]
I have updated to 6.5 version, then installed [Performant Translations](https://wordpress.org/plugins/performant-translations/) plugin and it has automatically created "l10n.php" files from ".mo" translations for my custom theme and for qtranslate-xt plugin.
The performance impact is good, no issues found for now.
If I'll find some issues, gonna post here.
